### PR TITLE
[MIG] hr_timesheet_sheet: Migration to 15.0

### PR DESCRIPTION
**Pull request by dochse**

### CODE REVIEW

**Review by dsolanki-initos — COMMENTED** [review]
Functionally test LGTM!

@dochse Can you please check failed checks?

**Comment by fuentes73** [comment]
Hi @dochse can you fix the pre-commit error?

**Review by gaikaz — APPROVED** [review]
Mig. changes LGTM
Please check pre-commit

File: hr_timesheet_sheet/__manifest__.py [hunk 1]
```diff
@@ -25,6 +25,11 @@
         "views/hr_employee_views.xml",
         "views/account_analytic_line_views.xml",
         "views/res_config_settings_views.xml",
-        "templates/assets.xml",
     ],
+    'assets': {
+        'web.assets_backend': [
+            '/hr_timesheet_sheet/static/src/js/backend.js',
+            '/hr_timesheet_sheet/static/src/scss/backend.scss'
```
Comment (gaikaz) [inline hr_timesheet_sheet/__manifest__.py:32]: Might be an issue after all. 
```suggestion
            'hr_timesheet_sheet/static/src/js/backend.js',
            'hr_timesheet_sheet/static/src/scss/backend.scss'
```

**Comment by JuanyDForgeflow** [comment]
@dochse are you going to continue with this?

**Comment by dsolanki-initos** [comment]
@dochse can you please fix the checked failed?

**Comment by dsolanki-initos** [comment]
@dochse can you please run pre-commit run -a command for fixing the GitHub checks

**Comment by dsolanki-initos** [comment]
@astirpe can you please review this PR?

**Comment by CRogos** [comment]
@dochse are you still working on this PR? 
I've created a [fork](https://github.com/CRogos/timesheet/tree/15.0-mig-hr_timesheet_sheet) of this PR and implemented the feedback of @gaikaz and tried to fix all pre-commit checks.
But there are several warnings where the string format variables are not used in the translation. I assume the translations need to be adapted, but I am not capable of these languages. 

_hr_timesheet_sheet/i18n/id.po:439: [W7968(po-msgstr-variables), ]  Translation string couldn't be parsed correctly using string%variables TypeError('not all arguments converted during string formatting')_

What is the procedure to continue with the translation issue and if @dochse is not active anymore? Shall I create a second PR #516 ?

**Comment by dochse** [comment]
> @dochse are you still working on this PR? I've created a [fork](https://github.com/CRogos/timesheet/tree/15.0-mig-hr_timesheet_sheet) of this PR and implemented the feedback of @gaikaz and tried to fix all pre-commit checks. But there are several warnings where the string format variables are not used in the translation. I assume the translations need to be adapted, but I am not capable of these languages.
> 
> _hr_timesheet_sheet/i18n/id.po:439: [W7968(po-msgstr-variables), ] Translation string couldn't be parsed correctly using string%variables TypeError('not all arguments converted during string formatting')_
> 
> What is the procedure to continue with the translation issue and if @dochse is not active anymore? Shall I create a second PR #516 ?

Im not actively working on it anymore, so you are free to do with this as you will

**Comment by pedrobaeza** [comment]
Supersede by #516

**Pull request closed without merging**